.gitignore

```diff
@@ -27,6 +27,7 @@ egs/*/*/wav*
 
 # tools related
 tools/chainer_ctc/
+tools/kaldi*
```

Comment:
If you use `tools/kaldi*`, we may not require `tools/kaldi/`, `tools/kaldi-io-for-python/`, etc.

Reply:
Yes, It will be better to set this to tools/kaldi*, especially when the kaldi installation is a symlink. Still doing some test, and upload this modification